htmltest/check-img_test.go

```diff
@@ -193,6 +193,42 @@ func TestImagePre(t *testing.T) {
 	tExpectIssueCount(t, hT, 0)
 }
 
+func TestImageUsemap(t *testing.T) {
+	// deals with valid usemap
+	hT := tTestFile("fixtures/images/usemapValid.html")
+	tExpectIssueCount(t, hT, 0)
+}
+
+func TestImageUsemapMapDoesNotExist(t *testing.T) {
+	// detects usemap pointing to a non-existent map
+	hT := tTestFile("fixtures/images/usemapMapDoesNotExist.html")
+	tExpectIssueCount(t, hT, 1)
```

Comment:
Alongside `tExpectIssueCount` please also use `tExpectIssue` to ensure the correct issue is caught

(`tExpectIssue(t, hT, "message here", 1)`)